sbi/inference/posteriors/sbi_posterior.py

```diff
@@ -165,9 +169,14 @@ def get_leakage_correction(
         
         Arguments:
             x: conditioning context for posterior $p(\theta|x)$. If None, uses self.x_o.
-        
             num_rejection_samples: number of samples used to estimate the factor
              (default: 10000).
+            force_update: if False, we do not re-evaluate the leakage correction if
+                the context is the same as self.x_o. If True, we always re-evaluate.
+                This is needed to enforce a new estimate of the leakage after later
+                rounds, i.e. round 2, 3, ...
```

Comment:
thats important! However, now if it is False by default we would not re-evaluate after round>1. Why not? Isnt that what we want?

Reply:
After each round, we call
```
self._neural_posterior.get_leakage_correction(
                    x=self._x_o,
                    force_update=True,
                    show_progressbar=self._show_progressbar,
                ),
``` 
So we evaluate after each round.

Reply:
Can you comment how this got resolved in the end?